src/css/addItem.module.css

```diff
@@ -0,0 +1,372 @@
+}
+
+.item-image {
+  border-radius: 5px;
+  margin-bottom: 8px;
+}
+
+.add-item-button {
+  display: block;
+  margin: 0 auto;
+  padding: 12px 25px;
+  background-color: #4CAF50;
+  color: white;
+  border: none;
+  border-radius: 5px;
+  font-size: 16px;
+  cursor: pointer;
+  transition: background-color 0.2s;
+}
+
+.add-item-button:hover {
+  background-color: #45a049;
+}
+
+/* Add Item Page Styling */
+.category-selector {
+  display: flex;
+  justify-content: center;
+  gap: 10px;
+  margin-bottom: 30px;
+  flex-wrap: wrap;
+}
+
+.category-button {
+  padding: 10px 15px;
+  background-color: #f1f1f1;
+  border: none;
+  border-radius: 20px;
+  cursor: pointer;
+  font-size: 14px;
+  transition: all 0.2s;
+}
+
+.category-button:hover {
+  background-color: #e0e0e0;
+}
+
+.category-button.selected {
+  background-color: #2196F3;
+  color: white;
+}
+
+.items-grid {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(150px, 1fr));
+  gap: 20px;
+  margin: 30px 0;
+}
+
+.item-card {
+  background-color: white;
+  border-radius: 8px;
+  padding: 15px;
+  text-align: center;
+  cursor: pointer;
+  transition: transform 0.2s, box-shadow 0.2s;
+  box-shadow: 0 2px 5px rgba(0, 0, 0, 0.1);
+}
+
+.item-card:hover {
+  transform: translateY(-5px);
+  box-shadow: 0 5px 10px rgba(0, 0, 0, 0.15);
+}
+
+.item-card.selected {
+  border: 3px solid #2196F3;
+  transform: translateY(-5px);
+  box-shadow: 0 5px 10px rgba(0, 0, 0, 0.15);
+}
+
+.item-card p {
+  margin-top: 8px;
+  font-size: 14px;
+}
+
+.button-container {
+  display: flex;
+  justify-content: center;
+  gap: 15px;
+  margin-top: 20px;
+}
+
+.back-button, .add-button {
+  padding: 10px 20px;
+  border: none;
+  border-radius: 5px;
+  cursor: pointer;
+  font-size: 16px;
+}
+
+.back-button {
+  background-color: #f1f1f1;
+  color: #333;
+}
+
+.add-button {
+  background-color: #2196F3;
+  color: white;
+}
+
+.back-button:hover {
+  background-color: #e0e0e0;
+}
+
+.add-button:hover {
+  background-color: #0b7dda;
+}
+
+.add-button:disabled {
+  background-color: #cccccc;
+  cursor: not-allowed;
+}
+
+
+
+/*Create Outfit board*/
+.outfitBoard{
+    background-color: black;
+    height:513px;
+    width:360px;
+
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    margin-top: 20px;
+    margin-bottom: 20px;    
+
+}
+
+
+.emptyOutfitText {
+  text-align: center;
+  font-size: 1.2rem;
+  margin-top: 2rem;
+  color: #777;
+}
+
+
+/*Item card*/
+
+.itemsGrid {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr); /* 4 equal-width columns */
+  gap: 18px; /* space between items (adjust as needed) */
+  justify-items: center; /* center items horizontally */
+  align-items: center;   /* center items vertically */
+}
+
+.itemCard {
+  width: 76px;
+  height: 94px;
+  background-color: white;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 10px;
+  position: relative
```

Comment:
Here, I would mention what this `position: relative` is for... `.itemCard__checkbox`